.gitignore

```diff
@@ -12,7 +12,10 @@ pom.xml.versionsBackup
 /Bundle/target/
 /lib/
 /Server/src/logs/
+/Server/src/config/config.xml
 /Server/src/test/resources/config/
+/Server/s.bat
+/Server/src/bin/c.bat
```

Comment:
Please roll back the changes here. Those ignored entries are your own files which will not be accepted into the OpenAS2 package.